packages/primitives/progress-bar/meta.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import type { TComponentConfig } from 'autoprops'
+import { Text } from '@primitives/text'
+import type { TProgressBar } from './src'
+
+export const config: TComponentConfig<TProgressBar, 'items'> = {
+  props: {
+    style: [{ minWidth: 100, minHeight: 40, backgroundColor: '#ffb3c7' }],
+    children: [<Text key="first">Me... a ProgressBar</Text>],
```

Comment:
Do we need children in ProgressBar?

Reply:
i would say, it give more flexibility on how it will be implemented, in bubble-next our progress bar is 2 divs, so i make sense for both of them to be inside primitive-progress-bar (or at least one), but if we would like to add animation, or somebody, or make it less of a "bar" and more of a circle, it make it possible